README.md

```diff
@@ -57,6 +58,8 @@ If this parameter is missing in `.conf` file, value is `undefined`.
 `updateMismatchedBaseImage`: Optional. When `true`, existing base images, which meet condition `mismatch > tolerance`, are updated with actual screenshot of web site, tolerance is then set to `0` and diff with mismatch is not saved to folder.
 Use only if base images exists. Parameter is not compatible with `prepareBaseImage`.
 
+`ignoreNothing`: Optional. When `true`, resemblejs covers 100% comparison of compared images and ignores nothing (e.g. similar colors) in found mismatch - every small change, every difficult pixel is resolved as a mismatch.
```

Comment:
What is default value of this settings
What is the behaviour when the value is set to false

Reply:
Added to detailed part of this flag.